tests/modules/ebi-metagenomics/checkm2/checkm2/main.nf

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { CHECKM2               } from '../../../../../modules/ebi-metagenomics/checkm2/checkm2/main.nf'
+include { CHECKM2_DOWNLOAD_DB   } from '../../../../../modules/ebi-metagenomics/checkm2/download_db/main.nf'
+
+workflow test_checkm2 {
+
+    CHECKM2_DOWNLOAD_DB()
+    meta = [ id:'test', single_end:false ]
+    input = [
+        meta,
+        file("./tests/modules/ebi-metagenomics/checkm2/checkm2/data/bins", checkIfExists: true)
```

Comment:
Check the linting updates branch. We added the nf-core test data fasta file in tests/config/test_data.config. Instead of our custom file in the data/bins folder, use the nf-core regular: file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)